src/compiler/data.ts

```diff
@@ -254,9 +255,9 @@ export namespace stack {
   /**
    * Add stacked data source, for feeding the shared scale.
    */
-  export function def(model: Model, stackCfg):VgData {
-    var dim = stackCfg.groupby;
-    var val = stackCfg.value;
+  export function def(model: Model, stackDef: StackDef):VgData {
```

Comment:
This doesn't create redundant variable name though.  

That said, I don't like the `stackDef` name.   But can't think of a better name.  
It combines some properties and config in the same object so I can't call it `StackConfig`.  

Any suggestion?